### ISSUE TITLE
Rough BoM cost estimate?

### ISSUE THREAD

**Issue by gabrielgrant** [DavidWenzler/reAM250#2]
Exciting project!

There is certainly unique value in having the design be open, but also curious how the projected costs compare to commercial/closed options. Do you have a rough estimate of total BoM cost and build time?

**Comment by SquishyBoiFrog** [comment]
I'm building one right now from the bill of materials and majority of material being from Germany I'm expecting this project to be anywhere from 15-50k. I'm currently building an estimate so other people can see what the will need before they dive head first into this project. I'll see if I can send a pdf with all of the costs of the material and where to source them. Looks like a lot of the materials will have to be machined or printed with a material that has a high heat deflection rate.

There are a lot of components that are pretty massive as one piece and would be wasteful to mill but I'm hoping I can make it out of a cheaper material or build it in section that can be bonded or welded together.

A lot of the material you have to make an account and have a business account in order to see the prices so we will se how that goes. Maybe later down the line we can fine similar resources in USA?

I'm only doing this as a side project and have no relation to this project just love building DIY printers.

**Comment by npeacock** [comment]
I'm interested in the same and see the same thing, like I managed to find the vacuum pump used on ebay for $3k but not sure what it costs new or how difficult to buy just one.  Very interested to hear what you come up with

**Comment by npeacock** [comment]
I made this google sheet with the prices I have found so far - https://docs.google.com/spreadsheets/d/1dQrnr1FCm8Troj53v6KMwRtA0BKFM3St_y-ShWR9q-g/edit?gid=749144357#gid=749144357 It was a bit tricky to export so @DavidWenzler  if you have the BOM in a CSV or similar that might be good to include in the documentation.

**Comment by SquishyBoiFrog** [comment]
I'm confused with on your list is what is the difference between your categories of "cost", "cost @ 1k", "price @1", and "price @ 1k", is this just a cost between buying in bulk? Also I'm noticing there isn't a category for CAD reference numbers. Do you think we can add that so we can reference the CAD and help other identify which items is what? We also might have to make a wiring diagram at one point for other user to be able to see where everything is plugged together as some people won't be able to understand the electronic key for routing wires.

**Comment by npeacock** [comment]
Lots to digest here:
Explaining the 'price' is unit price and 'cost' is unit price x qty.  Many of the line items repeat many times but I wanted to keep it compatible with the exported list. 

Yesterday when I tried to export from Fusion it was failing but it seemed to work today.  I have added a new tab for the fusion export "FusionExportedBom".  The part numbers don't seem to correlate to any of the fields in the "Artikelstückliste" also I'm not sure what the difference is between Artikelstückliste and Artikelsummenstückliste but they both seem to be BOMs und ich habe nur am bisschen Deutsch.  The overall part count from the Fusion export is 469 and the BOM total is 478 so looks like pretty much everything is in the model plus its creating line items for the subassemblies.

So I guess we will need to hear some clarifications from @DavidWenzler how he generated the current list that has the supplier part numbers, or is it just a manually created list.  If we could add the supplier part info in the Artikelstückliste they the Fusion model could be the source of truth and export out directly.

**Comment by SquishyBoiFrog** [comment]
Did you see this list: "docs/reAm250_BoM_en_v1_0_0.pdf" under docs, it seems to match everything in the assembly word for word and identifying numbers.

**Comment by npeacock** [comment]
Thanks, that's handy, even has the links.  I think the other list I was looking at just covers the electronics and wiring.  Total there was already around $26k and another $20k or so for the pumps, heat exchanger and laser control I was able to find parts for.  No one seems to have a price on the laser that I can find.

**Comment by SquishyBoiFrog** [comment]
I think this project is going to cost about 50k because the main supporting frame for the enclosure to 3d print it out of aluminum on PCBway is like 17k.

We got to find a cheaper solution.

**Comment by YGrahovski** [comment]
I think this will useful for your spreadsheet: https://www.edmundoptics.com/p/superscan-iv-10mm-1064nm-bundle/51480/#

**Comment by npeacock** [comment]
> I think this project is going to cost about 50k because the main supporting frame for the enclosure to 3d print it out of aluminum on PCBway is like 17k.
> 
> We got to find a cheaper solution.

Agree, thats my goal too, get something into the hobby price range.

How did you get that number?  PCB way does custom stuff right?  I thought the frame was all standard metals cut and welded.  Did put in as one big part and quote it?

**Comment by SquishyBoiFrog** [comment]
PCB way is a service I've hearing through out the DIY community for metal parts. You can go on their website and drop in your part that you want manufactured and choose the material. Seemed like a good idea until I realized that it supper expensive no matter the size of the part. I did put it as one part, what would be your idea to make it so we could make this part cheaper? Just rethinking we could possibly cut it into several pieces and have it machined which could possibly be a lot cheaper. The other thing we will have to figure out is if the chamber takes intense heat or any bar of pressure. If no, we could switch the material to a FDM material and save an arm and a leg. I have made an issue category on material selection.

**Comment by npeacock** [comment]
Right now my estimated cost is $52k + nLight_AFX-1000 + structural parts + pipes/misc.  I've reached out to nLight to see if I can get a budgetary number on the laser.  I can't imagine the structural parts or piping is going to be >$1-2k so in the noise at this point.

My POV, every reduction in hardware cost can improve access and lower the barrier to contribute.  I think the scanner and laser modules are essentially fixed in this build, the rest of the components are easier to replace.  Given that I'm not getting a BOM together for the other parts until I learn more about laser cost.

**Comment by npeacock** [comment]
Laser company got back, they won't share a price or do business with new businesses until they are at least a year old which was a new one to me.  They also don't seem to have any distributors, so that's a dead end.  Looking at random suppliers on AliExpress I'm seeing prices all over the place making it hard to know what compares.  For now, I don't see a way for an individual to acquire all the parts needed to participate with real hardware but the cost is at least 70k for the BOM.
